docs/proposals/note-creation-filepath-fallback-chain.md

```diff
@@ -0,0 +1,72 @@
+  - [Phase 4: Match `SAME_AS_ACTIVE_NOTE` behaviour](#phase-4-match-same_as_active_note-behaviour)
+  - [Phase 5: Match `WORKSPACE_ROOT` behaviour](#phase-5-match-workspace_root-behaviour)
+  - [Phase 6: `<home_dir>`](#phase-6-home_dir)
+- [Progression of Foam usage maturity](#progression-of-foam-usage-maturity)
+
+## Introduction
+
+This document aims to standardize the fallback chain that is used by Foam when determining where to create a new note.
+Currently, each command has its own fallback chain, which leads to inconsistent behaviour in Foam.
+
+This document will guide the development direction that each note-creating-command will eventually be modified to follow.
+
+Note that throughout this doc, `foam.newNoteDirectory` is used as a placeholder to denote a new Foam setting that we would add, as discussed in [this comment](https://github.com/foambubble/foam/issues/670#issuecomment-860184377).
+
+## Proposal: "New note" path resolution
+
+### Phase 1: `<workspace>/.foam/templates/new-note.md`
+
+If `<workspace>/.foam/templates/new-note.md` exists, parse it looking for `filepath`:
+
+* An absolute path in `filepath` in `<workspace>/.foam/templates/new-note.md` should just use that absolute path.
+* A relative path in `filepath` in `<workspace>/.foam/templates/new-note.md` should be relative to the workspace.
+
+### Phase 2: `<home_dir>/.foam/templates/new-note.md`
+
+If `<workspace>/.foam/templates/new-note.md` does not exist (or no `filepath` attribute is defined in it), fall back to `<home_dir>/.foam/templates/new-note.md`
+
+* An absolute path in `filepath` in `<home_dir>/.foam/templates/new-note.md` should just use that absolute path.
+* A relative path in `filepath` in `<home_dir>/.foam/templates/new-note.md` should result in:
+  * **Option A**: Relative to the home directory
+    * This behaviour matches `<workspace>/.foam/templates/new-note.md` behaviour
+  * **Option B**: Relative to `foam.newNoteDirectory`
+    * I suspect is what you'd expect if you've already defined `foam.newNoteDirectory`
+  * **Option C**: Both. Relative to `foam.newNoteDirectory`, only when `foam.newNoteDirectory` is defined. Otherwise, relative to `<home_dir>`
+    * Things get a little weird if you didn't have `foam.newNoteDirectory` defined, then later defined it.
+
+**TODO:** I haven't given enough thought to what is the best option here. My gut currently suspects that **Option C** is the right one.
+
+### Phase 3: `foam.newNoteDirectory`
+
+If `<home_dir>/.foam/templates/new-note.md` does not exist (or no `filepath` attribute is defined in it), fall back to `foam.newNoteDirectory`.
+
+`foam.newNoteDirectory` should be a filepath, matching the spec used for `filepath` in the templates:
+  * Consistency: with the behaviour of the existing template commands.
+  * Flexibility: Users could use variables like `$FOAM_TITLE` (and eventually other snippet variables) in the path.
+
+### Phase 4: Match `SAME_AS_ACTIVE_NOTE` behaviour
```

Comment:
I wonder if Phase 4, 5, 6 would be better described asa options of Phase 3. The same thing you do in Phase 2.